locale/en-us.yaml

```diff
@@ -1068,6 +1068,27 @@ language_name_unknown:
   other: Unknown
 language_unknown_options:
   other: Unknown options
+sample_yaml_perl:
+  other: |
+    scripts:
+    # This script uses a secret. Note that you can define your own secrets at
+    # https://platform.activestate.com/{{.Owner}}/{{.Project}}/scripts
+      - name: helloWorld
+        language: {{.Shell}}
+        value: echo ${secrets.user.world}
+      - name: activationMessage
+        language: perl
+        value: |
+          print "You are now in an 'activated state', this will give you a virtual environment to work in that doesn't affect the rest of your system.\n\n";
```

Comment:
I'm assuming Perl has a better solution than individual `print` statements for each line. Try to do it with a single print statement. 

The current implementation is awkward because of limitations we should no longer have.